### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by elbrujohalcon**

### CODE REVIEW

File: src/gadget_user.erl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+stop() -> application:stop(gadget_user).
+
+-spec start(atom(), any()) -> {ok, pid()} | {error, term()}.
+start(normal, _Args) -> {ok, self()}.
+
+-spec stop(atom()) -> ok.
+stop(_State) -> ok.
+
+%something(Wrong) -> Here.
+%% LOOK MA, A VERY VERY VERY LONG LINE WITH ALL CAPS AND REALLY NO MEANING AT ALL
```
Comment (gadgetci) [inline src/gadget_user.erl:26]: Line 26 is too long: "%% LOOK MA, A VERY VERY VERY LONG LINE WITH ALL CAPS AND REALLY NO MEANING AT ALL".

File: src/gadget_user_warning.erl [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-module(gadget_user_warning).
+
+-export([warning/1]).
+
+warning(Something) -> unused.
```
Comment (gadgetci) [inline src/gadget_user_warning.erl]: Warning: variable 'Something' is unused

Comment (gadgetci) [inline src/gadget_user_warning.erl]: Warning: missing specification for function warning/1

File: src/gadget_user.erl (hunk 1 above)
Comment (gadgetci) [inline src/gadget_user.erl:26]: Line 26 is too long: %% LOOK MA, A VERY VERY VERY LONG LINE WITH ALL CAPS AND REALLY NO MEANING AT ALL.

File: src/gadget_user_warning.erl [hunk 3]
```diff
@@ -0,0 +1,5 @@
+-module(gadget_user_warning).
+
+-export([warning/1]).
+
+warning(_Something) -> unused.
```
Comment (gadgetci) [inline src/gadget_user_warning.erl]: Warning: missing specification for function warning/1

File: src/gadget_user_warning.erl [hunk 4]
```diff
@@ -0,0 +1,5 @@
+-module(gadget_user_warning).
+
+-export([warning/2]).
+
+warning(Something, Else) -> unused.
```
Comment (gadgetci) [inline src/gadget_user_warning.erl:5]: Warning: variable 'Something' is unused

Comment (gadgetci) [inline src/gadget_user_warning.erl:5]: Warning: variable 'Else' is unused

Comment (gadgetci) [inline src/gadget_user_warning.erl:5]: Warning: missing specification for function warning/2

**Review by elvisci — CHANGES_REQUESTED** [review]
- Dependency 'sync' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'worker_pool' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by elbrujohalcon** [comment]
This PR is useless now 😢

**Pull request closed without merging**